Right now, an exception in a callback function is not handled properly.

Profiling

Documentation

Testing


